From 6fd73f9d3442c729a771d160430bd44fb003eb69 Mon Sep 17 00:00:00 2001
From: b_b <brunobergot@gmail.com>
Date: Wed, 27 Jul 2016 11:27:14 +0200
Subject: [PATCH] Fix for BitPaddedInt import error

This one fix an import error i get with latest python-mutagen 1.34.999+1301~0c8997f-0~ppa0~trusty from https://launchpad.net/~lazka/+archive/ubuntu/dumpingplace
---
 source/puddlestuff/audioinfo/_compatid3.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/source/puddlestuff/audioinfo/_compatid3.py b/source/puddlestuff/audioinfo/_compatid3.py
index 7d73e27..a7a599f 100644
--- a/source/puddlestuff/audioinfo/_compatid3.py
+++ b/source/puddlestuff/audioinfo/_compatid3.py
@@ -23,7 +23,8 @@
 import mutagen
 from mutagen._util import insert_bytes
 from mutagen.id3 import ID3, Frame, Frames, Frames_2_2, TextFrame, TORY, \
-    TYER, TIME, APIC, IPLS, TDAT, BitPaddedInt, MakeID3v1
+    TYER, TIME, APIC, IPLS, TDAT, MakeID3v1
+from mutagen.id3._util import BitPaddedInt
 
 SEPARATOR = ' / '
 
